From 38b743d5eface516f846b18d46f36620457bb138 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Tue, 21 Sep 2021 09:28:05 +0200
Subject: [PATCH] Fix installing README

c76426d262 renamed the README to README.md, but this change was never
reflected in CMakeLists.txt, thus the install target would fail
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7f56820e..e6d237ef 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -416,7 +416,7 @@ install(EXPORT MaliitPluginsTargets FILE MaliitPluginsTargets.cmake DESTINATION
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/MaliitPluginsConfig.cmake ${CMAKE_CURRENT_BINARY_DIR}/MaliitPluginsConfigVersion.cmake
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/MaliitPlugins)
 
-install(FILES INSTALL.local LICENSE.LGPL NEWS README
+install(FILES INSTALL.local LICENSE.LGPL NEWS README.md
         DESTINATION ${CMAKE_INSTALL_DATADIR}/doc/maliit-framework)
 
 if(enable-glib)
